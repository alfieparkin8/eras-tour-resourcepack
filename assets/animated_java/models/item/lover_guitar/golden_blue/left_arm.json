{"parent":"animated_java:item/lover_guitar/left_arm","textures":{"1":"minecraft:block/guitars/lover_blue","2":"minecraft:block/outfits/lover/golden_layer_1"}}