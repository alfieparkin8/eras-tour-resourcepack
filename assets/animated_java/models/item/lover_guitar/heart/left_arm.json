{"parent":"animated_java:item/lover_guitar/left_arm","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}