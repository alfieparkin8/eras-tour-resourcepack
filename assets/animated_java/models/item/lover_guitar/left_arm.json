{"textures":{"0":"minecraft:block/blank_hair"},"elements":[{"from":[6.5,-4,6],"to":[9.5,8,10],"rotation":{"angle":0,"axis":"y","origin":[5.5,-15.5,0]},"faces":{"north":{"uv":[9,13,9.75,16],"texture":"#0"},"east":{"uv":[8,13,9,16],"texture":"#0"},"south":{"uv":[10.75,13,11.5,16],"texture":"#0"},"west":{"uv":[9.75,13,10.75,16],"texture":"#0"},"up":{"uv":[9.75,13,9,12],"texture":"#0"},"down":{"uv":[10.5,12,9.75,13],"texture":"#0"}}}],"display":{"head":{"rotation":[0,180,0]}}}