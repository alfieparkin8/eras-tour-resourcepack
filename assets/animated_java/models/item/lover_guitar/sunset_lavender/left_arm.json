{
	"parent": "animated_java:item/lover_guitar/left_arm",
	"textures": {
		"2": "animated_java:item/lover_guitar/lover_lavender"
	}
}