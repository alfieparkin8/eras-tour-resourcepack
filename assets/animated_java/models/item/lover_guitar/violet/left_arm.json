{"parent":"animated_java:item/lover_guitar/left_arm","textures":{"2":"minecraft:block/outfits/lover/violet_layer_1"}}