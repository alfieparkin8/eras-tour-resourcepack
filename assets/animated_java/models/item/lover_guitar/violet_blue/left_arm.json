{"parent":"animated_java:item/lover_guitar/left_arm","textures":{"1":"minecraft:block/guitars/lover_blue","2":"minecraft:block/outfits/lover/violet_layer_1"}}