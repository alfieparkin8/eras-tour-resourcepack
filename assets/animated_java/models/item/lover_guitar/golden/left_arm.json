{"parent":"animated_java:item/lover_guitar/left_arm","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1"}}